javascript/implementation/Keyed.ts

```diff
@@ -79,6 +86,118 @@ export class Keyed<
         return result !== undefined;
     }
 
+    async reset(args?: {
+        toTime?: AsOf;
+        bundlerOrComment?: Bundler | string;
+        recurse?: boolean;
+    }): Promise<void> {
+        if (this.behavior === Behavior.PROPERTY) {
+            throw new Error(
+                "Cannot directly reset a property. " +
+                    "Calling reset on a Container will reset its properties."
```

Comment:
I suppose this is actually not true as of this pull request. Let me know if you want me to reword this temporarily for completeness.